nwbinspector/register_checks.py

```diff
@@ -121,9 +120,7 @@ def auto_parse(check_function, obj, result: Optional[InspectorMessage] = None):
     """Automatically fill values in the InspectorMessage from the check function."""
     if result is not None:
         auto_parsed_result = result
-        if auto_parsed_result.severity is None:  # For perfect consistency with not specifying
-            auto_parsed_result.severity = Severity.NO_SEVERITY
-        if auto_parsed_result.severity not in Severity:
+        if auto_parsed_result.severity not in [Severity.LOW, Severity.HIGH]:
```

Comment:
Would keeping `if auto_parsed_result.severity not in Severity` be OK?

Reply:
or `if isinstance(auto_parsed_result.severity, Severity)`

Reply:
Looks like `isinstance(auto_parsed_result.severity, Severity)` works well across platforms, though, I'll just swap to that, then. Good catch!